oyoyo/client.py

```diff
@@ -174,7 +181,115 @@ def connect(self):
                         sys.exit(1)
 
             if self.use_ssl:
```

Comment:
This is a pretty big indented block.

It may be worthwhile to refactor all the TLS code into a helper function.

Then, you can call the helper function only if self.use_ssl is True, thereby eliminating the need for a large indent block.